proto/aep-type/aep/type/phone_number.proto

```diff
@@ -117,5 +115,5 @@ message PhoneNumber {
   // field is normally only set in conjunction with an E.164 number. It is held
   // separately from the E.164 number to allow for short code extensions in the
   // future.
-  string extension = 3 [(buf.validate.field).string = { max_len: 40 }];
-}
\ No newline at end of file
+  string extension = 3 [(buf.validate.field).string = {max_len: 40}];
```

Comment:
One nit: it looks like this PR also makes some unrelated formatting changes. I think the protos were previously clang-formatted with Google style. Think we can preserve that, so that this PR only has contentful changes?

(making this an actual comment on the PR so that it has to be resolved before merging)

Reply:
Will revert the formatting changes, thanks.